tests/chart_tests/test_prometheus_config_configmap.py

```diff
@@ -305,3 +305,41 @@ def assert_relabel_config_for_non_auto_generated_namesaces(self, chart):
         ]
         assert metric_relabel_config_search_result[1]["replacement"] == "$1"
         assert metric_relabel_config_search_result[1]["target_label"] == "release"
+
+    def test_additional_scrape_jobs(self, kube_version):
+        doc = render_chart(
+            kube_version=kube_version,
+            show_only=self.show_only,
+            name="astronomer",
+            values={
+                "prometheus": {
+                    "additionalScrapeJobs": [
+                        {
+                            "job_name": "example-static-job",
+                            "static_configs": [{"targets": ["localhost:9090"]}],
+                        },
+                        {
+                            "job_name": "example-kubernetes-job",
+                            "kubernetes_sd_configs": [
+                                {
+                                    "role": "endpoints",
+                                    "namespaces": {"names": ["default"]},
+                                }
+                            ],
+                        },
+                    ]
+                }
+            },
+        )[0]
+        prometheus_config = yaml.safe_load(doc["data"]["config"])
+
+        found_static_job = False
+        found_kubernetes_job = False
+        for job in prometheus_config["scrape_configs"]:
+            if job["job_name"] == "example-static-job":
+                found_static_job = True
+            elif job["job_name"] == "example-kubernetes-job":
+                found_kubernetes_job = True
+
+        assert found_static_job, "Static job not found in rendered ConfigMap"
+        assert found_kubernetes_job, "Kubernetes job not found in rendered ConfigMap"
```

Comment:
Awesome job on the tests! I found a way to simplify the test you wrote while doing a more exact comparison between the inputs and outputs:
```suggestion
    def test_additional_scrape_jobs(self, kube_version):
        static_job = {
            "job_name": "example-static-job",
            "static_configs": [{"targets": ["localhost:9090"]}],
        }
        kubernetes_job = {
            "job_name": "example-kubernetes-job",
            "kubernetes_sd_configs": [
                {
                    "role": "endpoints",
                    "namespaces": {"names": ["default"]},
                }
            ],
        }
        doc = render_chart(
            kube_version=kube_version,
            show_only=self.show_only,
            name="astronomer",
            values={
                "prometheus": {
                    "additionalScrapeJobs": [
                        static_job,
                        kubernetes_job,
                    ]
                }
            },
        )[0]
        scrape_configs = yaml.safe_load(doc["data"]["config"])["scrape_configs"]

        assert static_job in scrape_configs
        assert kubernetes_job in scrape_configs
```